addons/main.tf

```diff
@@ -0,0 +1,32 @@
+data "aws_eks_cluster" "eks" {
+  name = "${var.cluster_name}"
+}
+
+data "aws_caller_identity" "current" {}
+
+provider "helm" {
+  kubernetes {
+    config_context = "${data.aws_eks_cluster.eks.arn}"
+  }
+  install_tiller = "${var.install_tiller}"
+  service_account = "${var.tiller_service_account}"
```

Comment:
Should we create the service account too?

Reply:
At some stage yes, the next part of "Terraformifying" the build would be to take the various manifests we apply `create.sh` and create them in Terraform.